### ISSUE TITLE
Not able to cpature screen recording

### ISSUE THREAD

**Issue by puneetamb** [rokuality/rokuality-js#12]
Hi Team,

getRecoding() is not working.

Error:
 ```
Failed to generate/retrieve video recording! See logs for details.

      17 |     _classCallCheck(this, ScreenException);
      18 |
    > 19 |     var _this = _possibleConstructorReturn(this, (ScreenException.__proto__ || Object.getPrototypeOf(ScreenException)).call(this, message));
      20 |
      21 |     _this.name = _this.constructor.name;
      22 |     return _this;
      
      at new ScreenException (../build/exceptions/ScreenException.js:19:120)
      at RokuScreen.handler (../build/driver/Screen.js:155:15)
      at RokuScreen.getRecording (../build/driver/Screen.js:141:33)
      at Object.<anonymous> (roku.test.js:124:7)
```

**Comment by rokualitysupport** [comment]
Can you include your server logs where the error is produced server side?

**Comment by puneetamb** [comment]
From where can I get server logs for rokuality ? I cannot see any documentation for that. Can you please share the steps that would help.

**Comment by rokualitysupport** [comment]
When you run a test in the UI builder, the logs can be saved from the help menu - agree it's not the most intuitive option. The next release exposes log retrieval by an api and also locks logs down to the test session as opposed to how they are simply stored now in a single output for all sessions.

**Comment by puneetamb** [comment]
I am not using the UI builder. I am using rokuality server to run the test and using `await screen.getRecording()` in my test to get the recording. However, it throws exception without detailing any logs and also cannot find any help menu when running rokuality server. 
Error thrown: 
`  Failed to generate/retrieve video recording! See logs for details.

      17 |     _classCallCheck(this, ScreenException);
      18 |
    > 19 |     var _this = _possibleConstructorReturn(this, (ScreenException.__proto__ || Object.getPrototypeOf(ScreenException)).call(this, message));
      20 |
      21 |     _this.name = _this.constructor.name;
      22 |     return _this;
      
      at new ScreenException (../build/exceptions/ScreenException.js:19:120)
      at RokuScreen.handler (../build/driver/Screen.js:155:15)
      at RokuScreen.getRecording (../build/driver/Screen.js:141:33)`

<img width="500" alt="Screenshot 2021-04-27 at 1 29 40 PM" src="https://user-images.githubusercontent.com/64578137/116206885-e2de9580-a75c-11eb-9798-fdab576ccc4b.png">

**Comment by puneetamb** [comment]
Can you please tell from where should I extract server logs ?

**Comment by rokualitysupport** [comment]
Correct, log capture via api/server is not yet available. It's coming in the next Rokuality release. Right now you can only view the logs during a UI session. From the main menu of the app, click "Run a Test", then enter in your test capabilities that would match in exactly the same fashion as those you are passing via the api's. This will start a debuggable UI test session with a help menu up top that will have a menu option to download the error logs.
